schema.py

```diff
@@ -0,0 +1,118 @@
+    closed_requests = Column(Integer, default=0) #The number of requests which the student opened that are now closed (filled)
+    gained_money = Column(Float, default=0) #The total amount of money this Student has gained (used for statistics)
+    approved = Column(Boolean, default = False) #There are certain actions an account is denied if unapproved
+
+
+    def __repr__(self):
+        ret = "<ID: %d, Opened Requests: %d, Closed Requests: %d, Money Gained: %d>" % (self.id, self.open_requests, self.closed_requests, self.gained_money)
+        return ret
+
+
+class Donor(Base, IdPrimaryKeyMixin, DateTimeMixin):
+    """
+    donors is the table for people who wish to make donations, represented by Donor objects
+    """
+    __tablename__= 'donors'
+
+    # Columns:
+    related_account = Column(Integer, nullable = False) #So we can easily locate the user information
+    donations_given = Column(Integer, default=0) #The number of requests this donor has closed (been the last one to donate to)
+    money_given = Column(Float, default=0) #The total amount of money this donor has donated
+
+    def __repr__(self):
+        ret = "<ID: %d, donations_given: %d, money_given: %d>" % (self.id, self.donations_given, self.money_given)
+        return ret
+
+class Request(Base, IdPrimaryKeyMixin, DateTimeMixin):
+    """
+    requests is the table which holds all the requests represented by Request objects
+    """
+    __tablename__='requests'
+
+    # Columns:
+    requested_by = Column(Integer) #who gave the request
+    amount_needed = Column(Float) #How much money do you need
+    description = Column(String) #Why the user needs money or requests for physical goods
+    amount_filled = Column(Float, default=0.0) #How much money has been raised
+    anon = Column(Boolean, default=True) #If 0 then the account does not wish to remain anonymous for actions on the website
+    filled = Column(Boolean, default=False) #After the request is filled, it no longer needs to be donated to
+    
+    def __repr__(self):
+        ret = "<Request ID: %d, Requested: %d, Filled: %d, Reason: %s>" % (self.id, self.amount_needed, self.amount_filled, self.description)
+        return ret
+
+    
+class Donation(Base, IdPrimaryKeyMixin, DateTimeMixin):
+    """
+    donations is the table of Donation objects
+    """
+    __tablename__='donations'
+
+    # Columns:
+    to_request = Column(Integer, nullable = False) #which request is being filled
+    from_donor = Column(Integer, nullable = False) #who is donating
+    amount_given = Column(Float) #how much money is being donated
+    charge_token = Column(String) #stripey?
+    anon = Column(Boolean, default = True) #Send anonymously  
+
+    def __repr__(self):
+        ret = '<Donation Key: %d, donated: %f, from account: %s' % (self.id, self.amount_given, self.from_donor)
+        return ret
+
+
+#####################################################
+Base.metadata.create_all(db.engine)

```

Comment:
So! All in all, this looks good. For the code that's there, my main critique is that it could use some Style Polish and a few more doc strings, but all in all, this seems like a damn fine start to me. 

Things to consider for next steps: 
1. Tests! 
2. I remark that, currently, none of your columns use _constraints_. That is: I don't see things explicitly labeled as foreign keys, for instance. These kinds of constraints are very good, and you should have them _eventually_ but I don't think putting them in place needs to block this PR right now.